actiontext/lib/generators/action_text/install/install_generator.rb

```diff
@@ -12,8 +12,7 @@ def install_javascript_dependencies
         run "rake app:update:bin"
 
         say "Installing JavaScript dependencies"
-        run "yarn add #{js_dependencies.map { |name, version| "#{name}@#{version}" }.join(" ")}",
-          abort_on_failure: true, capture: true
+        run "yarn add #{js_dependencies.map { |name, version| "#{name}@#{version}" }.join(" ")}"
```

Comment:
What could we do if we add something like:

```ruby
alphas, stables = js_dependencies.map { |name, version| "#{name}@#{version}" }.partition { |dependency| dependency.match?("alpha") }

say "Can't install these #{alphas.join(" ")}" if alphas.any? # What else can we do?
run "yarn add #{stables.join(" ")}" if stables.any?
```

Reply:
Sorry. I could not work on it. Is the next Rails 6.2 and future version always marked as `alpha` on the master branch? In that case, the warning would work without any problems.

> What else can we do?
Not sure but looking at `yarn link` or `yarn add <github_url>` for now. Will let you know when I have a solution for it. 🤞